kafka_utils/kafka_consumer_manager/commands/offset_advance.py

```diff
@@ -88,5 +90,13 @@ def run(cls, args, cluster_config):
                 "with --help option.", file=sys.stderr
             )
             raise
+        except UnknownMemberIdError:
+            print(
+                "Unable to unsubscribe group '{group_name}' from topic '{topic_name}'. \
```

Comment:
Just a small nit, otherwise LGTM for shipping. Fix'n'ship!
```suggestion
                "Unable to advance offsets for group '{group_name}' from topic '{topic_name}'. \
```